Sources/PubNub/PubNubConfiguration.swift

```diff
@@ -116,6 +118,8 @@ public struct PubNubConfiguration: Hashable {
   public var cipherKey: Crypto?
   /// If Access Manager (PAM) is enabled, client will use `authKey` on all requests
   public var authKey: String?
+  /// If Access Manager (PAM) is enabled, client will use  `token` instead of `authKey` on all requests
+  public var token: String?
```

Comment:
Is it possible to name this `authToken` to avoid confusion with any other type of `token`? `pamToken` would also work if `authKey` could be changed to `pamKey`

Reply:
For sure!